src/pages/ConfigurePatch.tsx

```diff
@@ -13,13 +15,31 @@ export const ConfigurePatch: React.FC = () => {
   });
   return (
     <PageWrapper>
-      {loading ? (
-        <PatchAndTaskFullPageLoad />
-      ) : error ? (
-        <div>{error.message}</div>
-      ) : (
-        <ConfigurePatchCore patch={data.patch} />
-      )}
+      {(function() {
```

Comment:
Can you move this logic out its kind of odd seeing a function and other variables declared within jsx.